src/theme/widgets.rs

```diff
@@ -98,7 +98,7 @@ impl<T: Spawn> Widgets for T {
             NodeBundle {
                 style: Style {
                     width: Px(500.0),
-                    justify_content: JustifyContent::Center,
+                    justify_content: JustifyContent::FlexStart,
```

Comment:
Fully deleting this line should work too I believe?

Reply:
Oh, right. On it

Reply:
[`2a6d9dc` (#259)](https://github.com/TheBevyFlock/bevy_quickstart/pull/259/commits/2a6d9dceef88144b4879a06ca64dcfc34f8c2208)

Also fixed the loading screen not being centered.